### PR TITLE
Improve .htaccess files to work with apache 2.4 w/o mod_access_compat

### PR DESCRIPTION
**Pull request by weberhofer**
Manttis issue 19662: Limesurvey does not work correctly with apache 2.4 without access_compat module

### CODE REVIEW

File: application/.htaccess [hunk 1]
```diff
@@ -1 +1,13 @@
-Deny from all
\ No newline at end of file
+<IfVersion < 2.4>
+    Order allow,deny
+    Deny from all
+</IfVersion>
```
Comment (Shnoulle) [inline application/.htaccess:4]: Why not

```
<IfVersion < 2.4>
    Order allow,deny
    Deny from all
</IfVersion>
<IfVersion >= 2.4>
        Require all denied
</IfVersion>
```

File: application/models/User.php [hunk 2]
```diff
@@ -245,8 +245,11 @@ public function getDateFormat()
      */
     public function getFormattedDateCreated()
     {
-        $dateCreated = $this->created;
-        $date = new DateTime($dateCreated);
+	$dateCreated = $this->created;
+	if (empty($dateCreated)) {
```
Comment (Shnoulle) [inline application/models/User.php:249]: Not related ;)

**Review by Shnoulle — COMMENTED** [review]
Why check `mod_access_compat` ?

Just check version

File: application/.htaccess (hunk 1 above)
Comment (weberhofer) [reply application/.htaccess:4]: I think, you are partially right. When users are on 2.4+, they can in every case use the `Require` setting.
But if they have additionally enabled `access_compat` they could have set something additional. It's not fully clear which setting is preferred (Apaches documentation says, one should not use require and Allow/Deny together) the compat settings should be overwritten, too.

I will update the pull-request.

**Review by Shnoulle — APPROVED** [review]
Seems OK

**Comment by weberhofer** [comment]
I'll prepare another pull request for the other issue with the non-set creation date of users tomorrow.

**Comment by Shnoulle** [comment]
Tested : :+1: have a 500 error before.

Thank you !

**Comment by Shnoulle** [comment]
@c-schmitz : can we merge it ?

**Pull request merged**